test/base/SpecBase.scala

```diff
@@ -41,4 +43,8 @@ trait SpecBase extends FreeSpec with MustMatchers with MockitoSugar with ScalaFu
         bind[AuthenticateActionProvider].to[FakeAuthenticateActionProvider]
       )
 
+  def injector: Injector = baseApplicationBuilder.injector
+
+  def appConfig = injector.instanceOf[AppConfig]
```

Comment:
Can we remove this? This is Injector and AppConfig was previously removed because in many tests, there were overriding module bindings but the tests used this injector which had different bindings from the tests. Instead in the tests where this is being used, they can just use the app's injector for that test.